infrastructure/code/backend_service_test.go

```diff
@@ -166,3 +168,85 @@ func TestSnykCodeBackendService_RunAnalysisSmoke(t *testing.T) {
 
 // todo analysis test limit files
 // todo analysis test severities
+
+func TestGetCodeApiUrl(t *testing.T) {
```

Comment:
nit: You could reduce duplication of the `inputList` generation and assertion between this and the test below, as it's the same logic.

You can write a table test for this with generated input and expected result. Not a blocker tho.